### DISCUSSION TITLE
Question: What's up with "New rendering" on roadmap?

### DISCUSSION THREAD

**Discussion by vitordelucca** [microsoft/WindowsAppSDK#426] [Q&A] [answered]
Is there any info on "New rendering" and "Smooth WinUI input & composition system experience" planned for Reunion 1.0?

https://github.com/microsoft/ProjectReunion/blob/main/docs/roadmap.md

**Comment by andrewleader** [comment, marked as answer]
Hey Victor, thanks for asking this, we're working on refining our roadmap this week and will let you know when we have more specific details to share!

Update: You can look at WinUI's roadmap which might have some more details on what's coming: https://github.com/microsoft/microsoft-ui-xaml/blob/main/docs/roadmap.md

If you aren't finding what you're looking for, you can file an issue in the WinUI microsoft-ui-xaml repo. Thanks :)

**Reply by float34** [reply]
@andrewleader Hi Andrew, any specific details to share?

**Reply by andrewleader** [reply]
Sorry for never updating you! You can look at WinUI's roadmap which might have some more details on what's coming: https://github.com/microsoft/microsoft-ui-xaml/blob/main/docs/roadmap.md

If you aren't finding what you're looking for, you can file an issue in the WinUI microsoft-ui-xaml repo. Thanks :)
